build/NuSpecs/Intellisense/Microsoft.UI.Xaml.xml

```diff
@@ -4,110 +4,206 @@
     <name>Microsoft.UI.Xaml</name>
   </assembly>
   <members>
-    <member name="T:Microsoft.UI.Xaml.Automation.Peers.AnimatedVisualPlayerAutomationPeer" />
+    <member name="T:Microsoft.UI.Xaml.Automation.Peers.AnimatedVisualPlayerAutomationPeer">
+      <summary>Exposes AnimatedVisualPlayer" types to Microsoft UI Automation.</summary>
```

Comment:
Why the single " (can be seen throughout the file)? (Looks to me like it was forgotten to be removed.)

Reply:
I have the same question @ranjeshj

Reply:
Looks like an issue with the intellisense pipeline. following up.

Reply:
This is fixed now.